micro_dl/preprocessing/resize_images.py

```diff
@@ -180,15 +179,9 @@ def resize_volumes(self, num_slices_subvolume=-1):
                                         self.scale_factor,
                                         self.input_dir,
                                         ff_path))
-                        cur_metadata = {'time_idx': time_idx,
-                                        'pos_idx': pos_idx,
-                                        'channel_idx': channel_idx,
-                                        'slice_idx': start_idx,
-                                        'file_name': op_fname}
-                        resized_metadata_list.append(cur_metadata)
 
         # Multiprocessing of kwargs
-        mp_utils.mp_rescale_vol(mp_args, self.num_workers)
+        resized_metadata_list = mp_utils.mp_rescale_vol(mp_args, self.num_workers)
```

Comment:
resized_metadata_list will be empty here as mp_utils.mp_rescale_vol doesn't return a meta dataframe

Reply:
I modified the behavior of mp_utils.mp_rescale_vol so it returns a list of dictionaries of the metadata rows